packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -99,6 +99,12 @@ export interface TransformConfig {
    * Such as sync configuration for appsync local support
    */
   ResolverConfig?: ResolverConfig;
+
+  /**
+   * List of custom transformer plugins
+   */
+  transformers?: string[];
+  warningESMessage?: boolean;
```

Comment:
Is there a need for these values in the transform config? If we moving away from using `ElasticsearchWarning` should we create a feature flag for that?

Reply:
These are the values that are getting passed here. Typescript complained about these missing fields in config when being used [here](https://github.com/yuth/amplify-cli/blob/5f674c843e45c6a230bee8bf26cef8bc2f40c8cd/packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts#L73-L76) and [here](https://github.com/yuth/amplify-cli/blob/5f674c843e45c6a230bee8bf26cef8bc2f40c8cd/packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts#L192-L195)